web/src/components/Transform.vue

```diff
@@ -119,6 +119,7 @@ v-layout.transform-component(row, fill-height)
         score-plot-tile(
             :width="600",
             :height="600",
+            :columns="dataset._source.columns",
```

Comment:
I'm deprecating `_source` soon, and you can get the same thing through an "officially sanctioned" property anway.  Please use `dataset.column.data`

Reply:
See b2a6c5ae5ec83409a3883c6f68bdc55803744f13.